content/fact/opensmtpd.md

```diff
@@ -10,6 +10,9 @@ daemon was needed for the base system of the operating system and no there
 were no suitable alternatives. The goals are simplicity, security and
 reliability with an acceptable [license](/fact/license/) (ISC).
 
+A major change in configuration was made with version 6.4, integrated into
+ OpenBSD 6.4.
+
 ```
```

Comment:
You are right, but why is this important? 

Would remove it and only focus on the tiny keypoints.

Users interested in OpenBSD should not be bothered with things were in the past.

Reply:
No pb! ;)